src/Sentry.Unity.Editor.iOS/SentryXcodeProject.cs

```diff
@@ -71,9 +74,37 @@ public void AddSentryFramework()
             _project.SetBuildProperty(unityFrameworkTargetGuid, "FRAMEWORK_SEARCH_PATHS", "$(inherited)");
             _project.AddBuildProperty(unityFrameworkTargetGuid, "FRAMEWORK_SEARCH_PATHS", "$(PROJECT_DIR)/Frameworks/");
 
+            _project.AddBuildProperty(unityFrameworkTargetGuid, "FRAMEWORK_SEARCH_PATHS", "$(PROJECT_DIR)/Frameworks/");
+            _project.AddBuildProperty(unityFrameworkTargetGuid, "FRAMEWORK_SEARCH_PATHS", "$(PROJECT_DIR)/Frameworks/");
+
+            _project.SetBuildProperty(mainTargetGuid, "DEBUG_INFORMATION_FORMAT", "dwarf-with-dsym");
+            _project.SetBuildProperty(unityFrameworkTargetGuid, "DEBUG_INFORMATION_FORMAT", "dwarf-with-dsym");
+
             _project.AddBuildProperty(mainTargetGuid, "OTHER_LDFLAGS", "-ObjC");
         }
 
+        public void AddBuildPhaseSymbolUpload(IDiagnosticLogger? logger)
+        {
+            if (MainTargetContainsSymbolUploadBuildPhase())
+            {
+                logger?.LogDebug("Build phase '{0}' already added.", SymbolUploadPhaseName);
+                return;
+            }
+
+            var mainTargetGuid = _project.GetUnityMainTargetGuid();
+            _project.AddShellScriptBuildPhase(mainTargetGuid,
+                SymbolUploadPhaseName,
+                "/bin/sh",
+                @"export SENTRY_PROPERTIES=sentry.properties
+if [[ ""$BITCODE_GENERATION_MODE"" == ""marker"" ]] ; then
```

Comment:
I belive `[[` is bash but this script is loading `/bin/sh`. Should we do `/bin/bash` instead?
Or rewrite the `if` not to use `[[`?